avaframe/in3Utils/cfgHandling.py

```diff
@@ -364,3 +365,59 @@ def filterCom1DFAThicknessValues(key, value, simDF):
     log.info('simulations for %s found with values: %s' % (key, simDF[allThNames]))
 
     return simDF
+
+
+def applyCfgOverride(cfgToOverride, cfgWithOverrideParameters, module, addModValues=False):
+    """ override configuration parameter values with the values provided in cfgWithOverrideParameters[modName_override]
+        also update the cfgWithOverrideParameters with the values for all parameters that are not
+        provided in the override parameters
+
+        Parameters
+        ----------
+        cfgToOverride: configparer object
+            configuration of module of interest
+        cfgWithOverrideParameters: configparser object
+            full configuration settings containing a section modName_override with parameter values
+            that should be overriden in the cfgToOverride
+        module
+            module of the cfgToOverride configuration
+
+        Returns
+        --------
+        cfgToOverride: configparser object
+            updated configuration of module
+        cfgWithOverrideParameters: configparser object
+            updated configuration of module
+    """
+
+    # get path of module
+    modPath = pathlib.Path(module.__file__).resolve().parent
+
+    # get filename of module
+    modName = str(pathlib.Path(module.__file__).stem)
+
+    # create list with parameters that become overridden
+    overrideParameters = cfgWithOverrideParameters['%s_override' % modName]
+    overrideKeys = [item for item in overrideParameters]
+
+    # loop through sections of the configuration of the module
+    foundKeys = []
+    for section in cfgToOverride.sections():
+        for key in cfgToOverride[section]:
+            if key in overrideKeys:
+                cfgToOverride.set(section, key, overrideParameters[key])
+                log.info('Override %s parameter: %s in section: %s with %s' % (modName, key, section, str(overrideParameters[key])))
+                foundKeys.append(key)
```

Comment:
I suggest switching this around. 
- loop through the sections (as it is now)
- but then loop through the overrideKeys
- use configParser.has_option() https://docs.python.org/3/library/configparser.html#configparser.ConfigParser.has_option  to check if this parameter exists in this section

Usually, there are a lot less overridekeys than keys in the toOverridecfg